packages/editor/src/store/selectors.js

```diff
@@ -1966,6 +1966,17 @@ export function isPostLocked( state ) {
 	return state.postLock.isLocked;
 }
 
+/**
+ * Returns whether post saving is locked.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean} Is locked.
+ */
+export function isPostSavingLocked( state ) {
+	return state.postSavingLock && state.postSavingLock.length > 0;
```

Comment:
When will `state.postSavingLock` be falsey?

Reply:
possibly before it is set? This is me writing defensive programs. I can remove this check.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/10649/commits/9117ace44cf980cc3d10583e53f574ddad56d55d